dds/DCPS/transport/framework/TransportReceiveStrategy_T.cpp

```diff
@@ -62,6 +62,15 @@ TransportReceiveStrategy<TH, DSH>::~TransportReceiveStrategy()
                  size));
     }
   }
+
+  for (int index = 0; index < RECEIVE_BUFFERS; ++index) {
```

Comment:
``` suggestion
 for (size_t index = 0; index < RECEIVE_BUFFERS; ++index) {
```

Reply:
This could result in a signed/unsigned comparison warning.  The type of `RECEIVE_BUFFERS` is int.

Reply:
It seems both are used in existing code for RECEIVE_BUFFERS, int and size_t